helm/charts/hydra/Chart.yaml

```diff
@@ -26,4 +26,4 @@ dependencies:
     version: 0.20.1
     condition: maester.enabled
     alias: hydra-maester
-    repository: file://../hydra-maester
+    repository: https://k8s.ory.sh/helm/charts
```

Comment:
I like this approach, clean and simple :) 
One thing thou, I would like to keep the second, file url as a comment. It is helpful to have it for local development (f.e if you want to modify the maester, and test that for hydra/oathkeeper)

Reply:
```suggestion
    # repository: file://../hydra-maester
    repository: https://k8s.ory.sh/helm/charts
```